### DISCUSSION TITLE
Unable to build Build/Lern/job/LernFlinkJobs/ service release tag release-5.3.1_RC1

### DISCUSSION THREAD

**Discussion by Arunkumarpodili** [Sunbird-Lern/Community#179] [Installation] [answered]
Hi @BharathwajShankar,

We are trying to build and deploy Sunbird on our environment.
while building the service Lerner service (Build/job/Lern/job/LernFlinkJob) we are facing some dependency issues. we are looking for your support to proceed further.

here is the error of the build job
[Lern_flinkjobs_error.txt](https://github.com/Sunbird-Lern/Community/files/13680626/Lern_flinkjobs_error.txt)

Thanks 
Arunkumar.p

**Comment by kumarks1122** [comment, marked as answer]
@Arunkumarpodili I noticed an extra space for a parameter in the mvn command from the log.
```
-DskipTests -DCLOUD_STORE_GROUP_ID=org.sunbird -DCLOUD_STORE_ARTIFACT_ID= cloud-store-sdk_2.12 -DCLOUD_STORE_VERSION=1.4.6
```
Kindly check in the jenkins config or parameters to check whether the extra space is there for `CLOUD_STORE_ARTIFACT_ID` in the beginning of it.
cc: @AmiableAnil

**Reply by srajasimman** [reply]
Hi @kumarks1122 

Thanks for your support. You suggestion work after removing extra space from cloud_store_artifact_id value.

Regards,
Rajasimman S
